go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/go-debos/fakemachine
+
+go 1.11
```

Comment:
how does this compare if we use a different version of go, say go 1.15 on the host (as per debian unstable) ?

Reply:
The 1.11 was chosen, since it's the one in Debian stable. I've only tested against Arch's 1.16. Chances are newer version might be required. Please check the official docs about the [go directive](https://golang.org/ref/mod#go-mod-file-go) - their wording if far better than any attempts of mine.

Note that the contents do not vary based on the go version.

Reply:
yup just read up on the config key; thanks for the link!